web/src/components/tasks/business-formation/review/ReviewBusinessSuffixAndStartDate.tsx

```diff
@@ -22,6 +23,15 @@ export const ReviewBusinessSuffixAndStartDate = (): ReactElement => {
     return `${name} ${suffix}`;
   };
 
+  const formattLawResponse = (willPracticeLaw: boolean | undefined): "Yes" | "No" | undefined => {
+    if (willPracticeLaw === true) {
+      return "Yes";
```

Comment:
Pebble - this content as well as the one on line 30 should be editable by the content team/live in the CMS

Reply:
Yep, thats a nice catch. Definielty copy pasted from the deleted ReviewWillPracticeLaw component but it should not have been that way in the first place.